docs/architecture.md

```diff
@@ -211,14 +211,31 @@ with a value of `prefill`.<br>
 *Type:* prefill-filter<br>
 *Parameters:* None<br>
 
-**KvCacheAwareScorer**<br>
-Scores based on real KV-cache state on vLLM. It is more accurate than either the SessionAffinity
-or PrefixCachePlugin, but requires extra computation and cycles to track the current cache state<br>
-*Type:* kvcache-aware-scorer<br>
-*Parameters:* Due to the sensitivity of the parameters of this plugin, the following
-environment variables are used to configure the scorer:<br>
-`KVCACHE_INDEXER_REDIS_ADDR` - the address of the Redis server used<br>
-`HF_TOKEN` - the Hugginface token to be used.<br>
+**PrefixCacheScorer**<br>
+The `prefix-cache-scorer` scorer scores a request based on the KV cache that each pod holds.
```

Comment:
nit: s/scorer scores/plugin scores/ or remove `scorer` completely.
nit: s/based on the KV cache that each pod holds/based on KV-cache localities/.